FHEM/00_SIGNALduino.pm

```diff
@@ -17,6 +17,7 @@ use Time::HiRes qw(gettimeofday);
 use Data::Dumper qw(Dumper);
 use Scalar::Util qw(looks_like_number);
 no warnings 'portable';
+use JSON;
```

Comment:
JSON ist laut Foren in den Abhängikeiten der Debian Installations  Pakete von FHEM gelistet.

Auf dem Travis Server hat das aber scheinbar nicht geklappt. 
Wir sollten vielleicht prüfen, ob das Modul geladen werden kann (EVAL) und dann die Optionen im Modul erst freischalten, wenn wir es haben. An gegebener Stelle können wir ja dann auch Fehlermeldungen ausgeben, dass das JSON modul benötigt wird.

Reply:
Eine Prüfung auf installiertes JSON sollten wir machen und dann auch ggf. in der Doku hinweisen!

Reply:
Eine Meldung bei get availableFirmware wäre vermutlich sehr hilfreich.

Reply:
Ich dachte vielleicht an einen Return wo man die Info erhält diese und jene Version ist verfügbar oder "die Versionen wurden abgerufen"

Reply:
Die werden in den Set Befehl flash eingebaut und erscheinen dadurch im dropdown